importexport/musicxml/musicxmlsupport.cpp

```diff
@@ -520,6 +524,10 @@ AccidentalType mxmlString2accidentalType(const QString mxmlName)
       map["natural-up"] = AccidentalType::NATURAL_ARROW_UP;
       map["flat-down"] = AccidentalType::FLAT_ARROW_DOWN;
       map["flat-up"] = AccidentalType::FLAT_ARROW_UP;
+      map["double-sharp-down"] = AccidentalType::SHARP2_ARROW_DOWN;
+      map["double-sharp-up"] = AccidentalType::SHARP2_ARROW_UP;
+      map["flat-flat-down"] = AccidentalType::FLAT2_ARROW_DOWN;
+      map["flat-flat-up"] = AccidentalType::FLAT2_ARROW_UP;
```

Comment:
These changes are in 3.5rc and in 3.x, so not an issue, but why then are they listed here?